src/Record.php

```diff
@@ -1294,7 +1323,7 @@ private function buildFields($create = true)
      */
     private function buildTableOptions()
     {
-        return " ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci";
+        return " ENGINE=InnoDB DEFAULT CHARSET=" . $this->character_set . " COLLATE=" . $this->collation;
```

Comment:
should use parameterized values for character_set and collation to prevent SQL injection?

```php
$this->values[] = $this->character_set;
$this->values[] = $this->collation;
return " ENGINE=InnoDB DEFAULT CHARSET=? COLLATE=?";
```

Reply:
Better safe than sorry :)